server.js

```diff
@@ -7,14 +7,9 @@
 var express = require("express");
 var bodyParser = require("body-parser");
 var path = require("path");
-<<<<<<< HEAD
```

Comment:
All deletions here have been ignored too - they are only deleting the <<<<<< things.